### PR TITLE
add section and semester tags button to the event creation/edition screen

### PR DESCRIPTION
**Pull request by unglazedstamp**

### CODE REVIEW

File: app/src/main/java/com/github/swent/echo/compose/event/EventTagEntry.kt [hunk 1]
```diff
@@ -35,9 +42,28 @@ fun EventTagEntry(
     onTagSelected: (tag: Tag) -> Unit,
     onTagDeleted: (tag: Tag) -> Unit
 ) {
-    var dialogVisible by remember { mutableStateOf(false) }
     Column(modifier = Modifier.fillMaxWidth().padding(EVENT_PADDING_BETWEEN_INPUTS)) {
         EventEntryName(stringResource(R.string.edit_event_screen_tags))
+        Row {
```
Comment (ntissieres) [inline app/src/main/java/com/github/swent/echo/compose/event/EventTagEntry.kt:47]: Some repetitions here, you could create a list of Pair<String, String> and then iterates over it

**Review by ntissieres — CHANGES_REQUESTED** [review]
Qualitative code, thanks for your work !  Just a little change to avoid repetition ;)

File: app/src/main/java/com/github/swent/echo/compose/event/EventTagEntry.kt (hunk 1 above)
Comment (unglazedstamp) [reply app/src/main/java/com/github/swent/echo/compose/event/EventTagEntry.kt:53]: Thanks a lot for the suggestions, I changed that.

**Review by ntissieres — DISMISSED** [review]
Thanks for the update ! Nice work ;)

**Review by ntissieres — APPROVED** [review]
Rebasing approval

**Pull request merged**